### DISCUSSION TITLE
突然用不了了“由于目标计算机积极拒绝，无法连接。”请问这是什么意思？

### DISCUSSION THREAD

**Discussion by kiradzS** [Jeric-X/SyncClipboard#105] [General]
Suddenly it doesn't work. "Connection is not possible because the target computer is actively rejecting it." What does that mean?

**Comment by Jeric-X** [comment]
It means "Connection is not possible because the target computer is actively rejecting it." 
建议优先检查网络联通性，排除网络问题后，如需帮助定位问题请按bug report的模板提供详细信息
